SmartDeviceLink/SDLSoftButtonReplaceOperation.m

```diff
@@ -56,11 +56,17 @@ - (void)start {
         SDLLogW(@"Soft button images are not supported. Attempting to send text-only soft buttons. If any button does not contain text, no buttons will be sent.");
 
         // Send text buttons if all the soft buttons have text
-        [self sdl_sendCurrentStateTextOnlySoftButtonsWithCompletionHandler:^{}];
+        __weak typeof(self) weakself = self;
+        [self sdl_sendCurrentStateTextOnlySoftButtonsWithCompletionHandler:^(BOOL success) {
+            if (!success) {
+                SDLLogE(@"Head unit does not support images and some of the soft buttons do not have text, so none of the buttons will be sent.");
```

Comment:
Isn't this redundant with the warning at L224?

Reply:
I don't think so. The warning on L224 can be benign if the head unit supports images but the library can't send the text only buttons because not all buttons have text (i.e. eventually the image + text buttons will be sent). This error is only printed if the head unit does not support images and the library can't send text-only buttons.